tests/prague/eip7692_eof_v1/eip7069_extcall/test_calls.py

```diff
@@ -470,3 +471,214 @@ def test_eof_calls_legacy_mstore(
         post=post,
         tx=tx,
     )
+
+
+@pytest.mark.parametrize(
+    ["opcode", "suffix"],
+    [
+        [Op.EXTCALL, [0, 0, 0]],
+        [Op.EXTDELEGATECALL, [0, 0]],
+        [Op.EXTSTATICCALL, [0, 0]],
+    ],
+    ids=["extcall", "extdelegatecall", "extstaticall"],
+)
+@pytest.mark.parametrize(
+    "destination_opcode, destination_suffix",
+    [[Op.REVERT, [0, 0]], [Op.INVALID, []]],
+    ids=["revert", "invalid"],
+)
+@pytest.mark.parametrize("destination_is_eof", [True, False])
+def test_eof_calls_revert_abort(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    sender: EOA,
+    opcode: Op,
+    suffix: list[int],
+    destination_opcode: Op,
+    destination_suffix: list[int],
+    destination_is_eof: bool,
+):
+    """Test EOF contracts calling contracts that revert or abort"""
+    env = Environment()
+
+    destination_contract_address = pre.deploy_contract(
+        Container.Code(destination_opcode(*destination_suffix))
+        if destination_is_eof
+        else destination_opcode(*destination_suffix)
+    )
+
+    caller_contract = Container.Code(
+        Op.SSTORE(slot_call_result, opcode(destination_contract_address, *suffix))
```

Comment:
```suggestion
        Op.SSTORE(slot_call_result, opcode(address=destination_contract_address))
```
The suffix can be removed by using the opcode keyword args.